giraffe/src/components/AnnotationLayer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {FunctionComponent, useMemo} from 'react'
+import {AnnotationLayerConfig, AnnotationLayerSpec, LayerProps} from '../types'
+import {getAnnotationsPositions} from '../utils/annotationData'
+
+interface AnnotationLayerProps extends LayerProps {
+  spec: AnnotationLayerSpec
+  config: AnnotationLayerConfig
+}
+
+const ANNOTATION_OVERLAY_DEFAULT_STYLE = {
+  width: '100%',
+  height: '100%',
+}
+
+export const AnnotationLayer: FunctionComponent<AnnotationLayerProps> = props => {
+  const {height, width, spec, xScale, yScale} = props
+  const annotationsPositions = useMemo(
+    () => getAnnotationsPositions(spec.annotations, xScale, yScale),
+    [spec.annotations, xScale, yScale]
+  )
+
+  return (
+    <svg style={{...ANNOTATION_OVERLAY_DEFAULT_STYLE}}>
```

Comment:
is there a way to override this? having `default` in the name implies there's a way to get around the default

Reply:
Yes, by de-structuring another set of styles after (to the right of) `ANNOTATION_OVERLAY_DEFAULT_STYLE`, the specific style properties will be overridden.

I did it this way as a clue on how to set configurable styling in the future.